docs/references/api/preferences.rst

```diff
@@ -292,3 +292,7 @@ With :ref:`RPC <functions>`, the preference is honored completely on the basis o
       "details": "The query affects 14 rows",
       "hint": null
   }
+
+.. note::
+
+  It is important for functions to return ``SETOF`` when called with ``max-affected`` preference. A violation of this would cause a :ref:`PGRST128 <pgrst128>` error.
```

Comment:
Just tested and this also works when the function returns `TABLE` , right? It should also be mentioned here and in the error message, I think.

Reply:
+1 on a test, also note:

> RETURNS TABLE also implies RETURNS SETOF.
> https://www.postgresql.org/docs/current/sql-createfunction.html